### PR TITLE
Use --attr=sql_backend for tavis

### PR DESCRIPTION
**Pull request by benrudolph**
#dsa-test-crush-day

### CODE REVIEW

File: corehq/apps/receiverwrapper/tests/test_app_id.py [hunk 1]
```diff
@@ -66,3 +65,9 @@ def _test_app_version(self, domain, form, build):
         self.assertEqual(build.version, 3)
         self.assertEqual(get_version_from_build_id(domain, form.build_id),
                          build.version)
+
+
+@use_sql_backend
+class TestAppIdSQL(TestAppId):
+    pass
+
```
Comment (dimagimon) [inline corehq/apps/receiverwrapper/tests/test_app_id.py]: W391 blank line at end of file

File: corehq/ex-submodules/couchforms/tests/test_archive.py [hunk 2]
```diff
@@ -125,3 +127,5 @@ def testPublishChanges(self):
                 xform.unarchive()
         self.assertEqual(1, len(change_context.changes))
         self.assertEqual(change_context.changes[0].id, xform.form_id)
+
+
```
Comment (dimagimon) [inline corehq/ex-submodules/couchforms/tests/test_archive.py]: W391 blank line at end of file

File: corehq/form_processor/tests/test_kafka.py [hunk 3]
```diff
@@ -1,21 +1,105 @@
...
+        form = submit_case_blocks(ledger_blocks, self.domain)[0]
+
+        # submit duplicate
+        with process_kafka_changes(self.ledger_pillow):
+            with process_couch_changes('DefaultChangeFeedPillow'):
+                dupe_form = submit_form_locally(form.get_xml(), domain=self.domain)[1]
+                self.assertTrue(dupe_form.is_duplicate)
+
+
+        # confirm republished
```
Comment (dimagimon) [inline corehq/form_processor/tests/test_kafka.py]: E303 too many blank lines (2)

**Comment by benrudolph** [comment]
sorry simon 😞

**Review by snopoke — APPROVED** [review]

**Comment by benrudolph** [comment]
Before
screen shot 2017-02-25 at 6 29 23 pm
After
screen shot 2017-02-25 at 6 29 10 pm

**Pull request merged**